webapp/public/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
```

Comment:
Let's remove generated page please

Reply:
I'll remove comments only since this is a necessary file